examples/parquet_analytics_demo.ipynb

```diff
@@ -0,0 +1,470 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d76d4d85",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Configure container access to Azure storage\n",
+    "az <- create_azure_login(tenant=\"9ce70869-60db-44fd-abe8-d2767077fc8f\", host=\"https://storage.azure.com/\")\n",
+    "token <- az$token$credentials$access_token\n",
+    "adls_ep <- adls_endpoint(\"https://pitestdatasa.dfs.core.windows.net/\", token=token)\n",
+    "container <- storage_container(adls_ep, \"gold\")\n",
+    "\n",
+    "# Initialize some tempfiles to buffer data into\n",
+    "phdi_conditions = tempfile()\n",
+    "phdi_organizations = tempfile()\n",
+    "phdi_compositions = tempfile()\n",
+    "phdi_patients = tempfile()\n",
+    "\n",
+    "# Perform storage download, copying data into the temp files\n",
+    "storage_download(container, \"Condition.parquet\", phdi_conditions)\n",
+    "storage_download(container, \"organization.parquet\", phdi_organizations)\n",
+    "storage_download(container, \"composition.parquet\", phdi_compositions)\n",
+    "storage_download(container, \"patient.parquet\", phdi_patients)\n",
+    "\n",
+    "# Notice that we can read-in only a subset of columns if we so choose\n",
+    "# i.e. It's not necessary to fully read the file to memory\n",
+    "conditions = read_parquet(phdi_conditions, col_select = c(\"patient_id\", \"onset_date\"))\n",
+    "organizations = read_parquet(phdi_organizations)\n",
+    "compositions = read_parquet(phdi_compositions)\n",
+    "patients = read_parquet(phdi_patients)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a83320d1",
+   "metadata": {},
+   "source": [
+    "We've now created local copies of all data that maps into the ECR schema--we have resources for Patients, Conditions, Organizations, and Compositions (between patients and organizations). Importantly, we've stored all of this data into `tempfiles` which will persist _only_ as long as our current R session. That way, when we're finished with analysis, the files will be cleared, leaving no chance for PII to linger. We'll use these file for the remainder of our analysis demo, noting that pulling files down this way allows analysts to experiment freely on data without any risk of corrupting the source of truth.\n",
+    "\n",
+    "Before getting into some demo analytics problems, we need to do a bit of data modification and format. If we try to visualize the `patient` data frame, we'll get the following error:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1d4e9a0c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "head(patients)"
```

Comment:
Nit: it would be nice to put a comment here to make it painfully clear this is expected to result in an error.  That fact is already clear from the description, but it might help to have it in the comments for the tl;dr scenario.